src/code/z_effect_soft_sprite_old_init.c

```diff
@@ -566,7 +566,7 @@ void EffectSsHahen_Spawn(GlobalContext* globalCtx, Vec3f* pos, Vec3f* velocity,
  *     - due to how life is implemented it is capped at 200. Any value over 200 is accepted, but the fragment will
  *       only live for 200 frames
  */
-#ifdef NON_MATCHING
+//#ifdef NON_MATCHING
 void EffectSsHahen_SpawnBurst(GlobalContext* globalCtx, Vec3f* pos, f32 burstScale, s16 unused, s16 scale,
```

Comment:
Oh right forgot about that

I randomly stumbled on that don't know why I tried removing the NON-MATCHING but it works without it, that was weeks ago so I forgot about the whole thing because it seemed weird, if it was matching why was it marked non-matching?